src/server/logs/logs.go

```diff
@@ -169,9 +169,21 @@ func compileRequest(request *logs.GetLogsRequest) (string, func(*logs.LogMessage
 		case *logs.UserLogQuery_Pipeline:
 			pipeline := query.Pipeline.Pipeline
 			project := query.Pipeline.Project
-			return fmt.Sprintf(`{app="pipeline",suite="pachyderm",pipelineProject=%q,pipelineName=%q}`, project, pipeline),
-				func(lm *logs.LogMessage) bool {
-					return true
+			return fmt.Sprintf(`{app="pipeline",suite="pachyderm",container="user",pipelineProject=%q,pipelineName=%q}`, project, pipeline),
+				func(msg *logs.LogMessage) bool {
+					if msg.GetPpsLogMessage().GetUser() {
+						return true
+					}
+					ff := msg.GetObject().GetFields()
+					if ff != nil {
+						v, ok := ff["user"]
```

Comment:
This is probably closer to what we do now.  Do we have a way to distinguish between "all pipeline logs" and "just my logs"?  If not, we should add the other one to the protocol.